tensornetwork/__init__.py

```diff
@@ -1,4 +1,4 @@
 from __future__ import absolute_import
 from tensornetwork.tensornetwork import TensorNetwork, Node, Edge
 from tensornetwork.ncon_interface import ncon, ncon_network
-
+from tensornetwork.contractors import *
```

Comment:
I think it'd be good to avoid wildcard imports since they can cause name collisions. Safer to list them all explicitly here. We only have a few :-)

Reply:
Done